policy/scan/local_scanner.go

```diff
@@ -595,7 +595,11 @@ func (s *localAssetScanner) run() (*AssetReport, error) {
 		return ar, err
 	}
 	s.ProgressReporter.Score(report.Score.Rating().Letter())
-	s.ProgressReporter.Completed()
+	if report.Score.Rating().Letter() == "U" {
```

Comment:
I think, this is different. It is not an error, the asset matched a policy, but didn't get scored.

I used this policy to provoke this:
```
  - asset_filter:
      query: platform.name == "k8s-deployment"
    scoring_queries:
    data_queries:
```

@preslavgerchev In which situation did you see the problem with the progress bar?

Reply:
I tested this with the `Kubernetes Best Practices by Mondoo` policy, however it had most of its controls disabled or ignored, leading to the situation you described above (matching filter, no queries).

i can provide the exact policy (with ignored queries) if needed

Reply:
I am not 100% sure I understand what we are solving here then. In case policies applied it was scored. Feels like the issue is with the policy itself.

Reply:
Preslav showed me the issue. I agree that we need to fix the overall progress, but reading the score itself feels wrong. The progress bar should not care about the rating at all. We can move forward with this approach as mitigation but we need a better solution for that handling going forward.

Reply:
Follow-up is https://github.com/mondoohq/cnspec/pull/439